doc/apimanager-reference.md

```diff
@@ -69,22 +69,23 @@ Generated using [github-markdown-toc](https://github.com/ekalinin/github-markdow
+| ImageStreamTagImportInsecure    | `imageStreamTagImportInsecure`    | bool | No | `false` | Set to true if the server may bypass certificate verification or connect directly over HTTP during image import                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| ImagePullSecrets                | `imagePullSecrets`                | \[\][corev1.LocalObjectReference](https://v1-17.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#localobjectreference-v1-core) | No | `[ { name: "threescale-registry-auth" } ]` | List of image pull secrets to be used on the managed DeploymentConfigs ServiceAccounts. See [imagePullSecrets field in K8s ServiceAccount documentation](https://v1-17.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#serviceaccount-v1-core) for details on Image pull secrets. If not specified, `threescale-registry-auth` is used. Secret names that contain `dockercfg-` or `token-` anywhere in part of its name cannot be specified. If an update to this attribute is performed the corresponding DeploymentConfig pods have to be redeployed by the user to make the changes effective |
+| ResourceRequirementsEnabled     | `resourceRequirementsEnabled`     | bool | No | `true` | When true, 3Scale API management solution is deployed with the optimal resource requirements and limits. Setting this to false removes those resource requirements. ***Warning*** Only set it to false for development and evaluation environments. When set to `true`, default compute resources are set for the APIManager components. See [Default APIManager components compute resources](#Default-APIManager-components-compute-resources) to see the default assigned values                                                                                                                                |
+| DisableApicastServiceReconciler | `disableApicastServiceReconciler` | bool | No | `false` | When false, 3scale API management solution will reconcile apicast services as normal. Setting this to true will stop 3scale from reconcilin apicast service ports                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
This is part of the `spec` and you have defined an annotation which is part of `metadata`. It need it's own section that you need to  create as it is the first time we document an annotation for the APIManager